src/Microsoft.AspNetCore.Mvc.Core/ContentResult.cs

```diff
@@ -67,13 +67,17 @@ public override Task ExecuteResultAsync(ActionContext context)
 
             if (Content != null)
             {
-                var bufferingFeature = response.HttpContext.Features.Get<IHttpBufferingFeature>();
-                bufferingFeature?.DisableResponseBuffering();
+                var data = resolvedContentTypeEncoding.GetBytes(Content);
```

Comment:
@Tratcher set me straight on this. Since `ContentResult` only uses strings we can compute the byte count up front, and then using a 'rolling' buffer to iterate through the string. So lets do that.